python/example_code/ecs/create_cluster.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# This file is licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+# OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+
+import boto3
+try:
+  ecs_client = boto3.client('ecs', aws_access_key_id='ACCESS_KEY',
+                                   aws_secret_access_key='SECRET_KEY',
+                                   region_name='REGION')
```

Comment:
You do not need to add the key_id, acess_key, or region since that is added differently depending on how to establish credentials for your environment.

Often we just use environment variables or the .aws/credentials file.